bindings/capacitorjs/src/definitions.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { EventType, AccountManagerOptions } from './types';
+export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
+    messageHandlerNew(messageOptions: AccountManagerOptions): Promise<{
+        messageHandler: number;
+    }>;
+    sendMessage(messageOptions: {
+        message: string;
+        handler: number;
+    }): Promise<{
+        result: string;
+    }>;
+    listen(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
+    }, callback: (message: {
+        error: {
+            cause: unknown;
+        };
+        result: string;
+    }) => void): Promise<void>;
+    clearListeners(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
+    }, callback: (message: {
+        error: {
+            cause: unknown;
+        };
+        result: string;
+    }) => void): Promise<void>;
+    destroy(options: {
+        messageHandler: number;
+    }): Promise<void>;
+}
```

Comment:
I don't think you need `options` here

Reply:
if you delete it will not work, it's the Capacitor plugin API way to do it. I think we talk about this on the call